server/api.py

```
@@ -78,4 +105,37 @@ async def create_chat_completion(request: ChatCompletionRequest):
                 headers={"Cache-Control": "no-cache"},
             )
     except Exception as e:
+        logger.exception("[CODE_INTERPRETER] Error in create_chat_completion")
         raise HTTPException(status_code=500, detail=str(e))
```

Ruff (0.14.11)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)